packages/twenty-front/src/modules/workspace/components/WorkspaceInviteTeam.tsx

```diff
@@ -69,7 +69,7 @@ type FormInput = {
 
 export const WorkspaceInviteTeam = () => {
   const { enqueueSnackBar } = useSnackBar();
-  const [sendInviteLink] = useSendInviteLinkMutation();
+  const { sendInvitation } = useInviteUser();
```

Comment:
example here: we are actually making WorkspaceInviteTeam re-render on invitations state change only to get the sendInvitation method. This is only correct if we make sure that useInviteUser does not re-render!